astropy/modeling/fitting.py

```diff
@@ -898,10 +892,10 @@ def __call__(self, model, x, y, z=None, weights=None, **kwargs):
         if len(model) == 1:
             model_set_axis = None
         else:
-            if (not hasattr(self.fitter, 'supports_masked_input') or
-                    self.fitter.supports_masked_input is not True):
-                raise ValueError("{} cannot fit model sets with masked "
-                                 "values".format(type(self.fitter).__name__))
+            if not hasattr(self.fitter, 'supports_masked_input') or \
+               self.fitter.supports_masked_input is not True:
```

Comment:
Not sure what happened here but this diff has nothing to do with f-string.
```suggestion
            if (not hasattr(self.fitter, 'supports_masked_input') or
                    self.fitter.supports_masked_input is not True):
```

Reply:
This was probably because I made this branch before #13137 was merged, and somehow it got messed up during the rebase.